server/neptune/workflows/internal/activities/terraform.go

```diff
@@ -215,6 +212,18 @@ func (t *terraformActivities) TerraformApply(ctx context.Context, request Terraf
 	return TerraformApplyResponse{}, nil
 }
 
+type TerraformCloseJobRequest struct {
+	JobID string
+}
+
+func (t *terraformActivities) TerraformCloseJob(ctx context.Context, request TerraformCloseJobRequest) error {
+	err := t.StreamHandler.Close(ctx, request.JobID)
+	if err != nil {
+		logger.Error(ctx, errors.Wrapf(err, "closing job with id: %s", request.JobID).Error())
+	}
+	return err
+}
```

Comment:
Hmm why is this in the terraform activities file? Personally would be down to separate it just because this file is already big and this technically doesn't need to be aware of the "Terraform" concept.